openreview/venue_request/process/deployProcess.py

```diff
@@ -390,4 +390,18 @@ def process(client, note, invitation):
             signatures = ['~Super_User1']
         )
 
-        client.post_invitation(matching_invitation)
\ No newline at end of file
+        client.post_invitation(matching_invitation)
```

Comment:
Looks like the comment invitation's reply.readers are updated only when we setup the paper matching, it should be outside of the if.

Reply:
Fixed.